region/brinstar/pink/Pink Brinstar Hopper Room.json

```diff
@@ -534,37 +794,58 @@
       "note": "Using a power bomb against the left wall will kill the enemies and trigger the elevator."
     },
     {
-      "link": [2, 2],
-      "name": "Leave With Runway",
-      "requires": [],
-      "exitCondition": {
-        "leaveWithRunway": {
-          "length": 6,
-          "openEnd": 1
-        }
-      }
+      "link": [3, 1],
+      "name": "Dead Sidehoppers",
+      "requires": [
+        {"obstaclesCleared": ["A"]}
+      ]
     },
     {
-      "link": [2, 2],
-      "name": "Safe Kill",
+      "link": [3, 1],
+      "name": "Base",
       "requires": [
-        {"enemyKill": {
-          "enemies": [
-            ["Sidehopper", "Sm. Sidehopper", "Sm. Sidehopper"]
-          ],
-          "explicitWeapons": ["PowerBomb", "ScrewAttack", "Wave"]
-        }}
-      ],
-      "clearsObstacles": ["A"],
-      "note": "With access to the right side of the room, it's possible to kill the enemies safely with Wave or a PB or screw attack once on the other side."
+        {"or": [
+          {"and": [
+            "canCarefulJump",
+            "canCameraManip"
+          ]},
```

Comment:
How does this work?

Reply:
I thought I was setting them up well from the right, but looks like it wasnt a thing.